src/FSharpLint.Core/Application/Lint.fs

```diff
@@ -468,13 +468,13 @@ module Lint =
                 try
                     // Use Microsoft.Build.Construction.SolutionFile for modern solution parsing
                     let solutionFile = SolutionFile.Parse(solutionFilePath)
-                    
+
                     let projectsInSolution =
                         solutionFile.ProjectsInOrder
-                        |> Seq.filter (fun project -> 
-                            project.ProjectType = SolutionProjectType.KnownToBeMSBuildFormat && 
+                        |> Seq.filter (fun project ->
+                            project.ProjectType = SolutionProjectType.KnownToBeMSBuildFormat &&
                             project.RelativePath.EndsWith(".fsproj"))
-                        |> Seq.map (fun project -> 
+                        |> Seq.map (fun project ->
```

Comment:
@xperiandri please let's not include whitespace changes in the PR to avoid git-blame noise